recipes/matio/all/conanfile.py

```diff
@@ -0,0 +1,98 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+
+class MatioConan(ConanFile):
+    name = "matio"
+    license = "BSD-2-Clause"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://sourceforge.net/projects/matio/"
+    description = "Matio is a C library for reading and writing binary MATLAB MAT files."
+    topics = ("conan", "matlab", "mat-file", "file-format", "hdf5")
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake"
```

Comment:
You're not using the `cmake_find_package` generator.
By using generated the generated `FindXXX` cmake modules and if you are using the "official upstream` cmake modules, then it should work transparantly.

Reply:
Not sure what I am expected to adapt.